src/native/managed/cdacreader/src/Legacy/SOSDacImpl.cs

```diff
@@ -27,8 +27,8 @@ internal sealed unsafe partial class SOSDacImpl
       ISOSDacInterface11, ISOSDacInterface12, ISOSDacInterface13, ISOSDacInterface14, ISOSDacInterface15
 {
     private readonly Target _target;
-    private readonly TargetPointer _stringMethodTable;
-    private readonly TargetPointer _objectMethodTable;
+    private readonly Lazy<TargetPointer> _stringMethodTable;
+    private readonly Lazy<TargetPointer> _objectMethodTable;
```

Comment:
Let's add a comment here as to why we're being lazy. This seems like a nuance in the current cDAC implementation in CoreCLR rather than a fundamental design choice.